src/axom/sidre/core/MFEMSidreDataCollection.hpp

```diff
@@ -0,0 +1,556 @@
+// Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
```

Comment:
@joshessman-llnl  -- are there any notable diffs between this and the original mfem versions?

Reply:
Other than the format changes and the `VerifyMeshBluePrint` implementation, I don't believe so.